doc/src/site/sphinx/deployment/load_mojo_pipeline.rst

```diff
@@ -0,0 +1,161 @@
+Importing MOJO Pipelines from Driverless AI
+-------------------------------------------
+
+MOJO scoring pipeline artifacts, created in Driverless AI, can be used in Spark to carry out predictions in parallel
+using the Sparkling Water API. This section shows how to load and run predictions on the MOJO scoring pipeline in
+Sparkling Water
+
+**Note**: Sparkling Water is backwards compatible with MOJO versions produced by different Driverless AI versions.
+
+One advantage of scoring the MOJO artifacts is that ``H2OContext`` does not have to be created if you only want to
+run predictions on MOJOs using Spark. This is because the scoring is independent of the H2O run-time.
+
+Requirements
+~~~~~~~~~~~~
+
+In order to use the MOJO scoring pipeline, Driverless AI license has to be passed to Spark.
+This can be achieved via ``--jars`` argument of the Spark launcher scripts.
+
+**Note**: In Local Spark mode, please use ``--driver-class-path`` to specify path to the license file.
+
+We also need Sparkling Water distribution which can be obtained from `H2O Download page <https://www.h2o.ai/download/>`__.
+After we downloaded the Sparkling Water distribution, extract it and go to the extracted directory.
+
+Loading and Score the MOJO
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, start the environment for your language with Driverless AI license. There are two variants. We can use
+Sparkling Water prepared scripts which put required dependencies on the Spark classpath or we can use Spark directly
+and add the dependencies manually.
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: bash
+
+            ./bin/spark-shell --jars license.sig,jars/sparkling-water-assembly_2.11-SUBST_SW_VERSION-all.jar
+
+        .. code:: bash
+
+            ./bin/sparkling-shell --jars license.sig
+
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: bash
+
+            ./bin/pyspark --jars license.sig --py-files py/build/dist/h2o_pysparkling_SUBST_SPARK_MAJOR_VERSION-SUBST_SW_VERSION.zip
+
+        .. code:: bash
+
+            ./bin/pysparkling --jars license.sig
+
+
+At this point, you should have Spark interactive terminal where we can carry out predictions.
+For productionalizing the scoring process, we can use the same configuration,
+except instead of using Spark shell, we would submit the application using ``./bin/spark-submit``.
+
+Now Load the MOJO as:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            import ai.h2o.sparkling.ml.models.H2OMOJOPipelineModel
+            val settings = H2OMOJOSettings()
```

Comment:
What about using some non default values of `H2OMOJOSettings` in the example.

Reply:
Good idea!